waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -952,17 +959,35 @@
    :periodSeconds health-check-interval-secs
    :timeoutSeconds 1})
 
+(defn- envoy-sidecar-env?
+  "Check environment flags to see if envoy sidecar should be enabled.
+   Used to build other predicate functions by providing defaults."
+  [env proxy-default tls-default]
+  (let [reverse-proxy? (-> env (get "REVERSE_PROXY" proxy-default) utils/string-yes?)
+        strict-tls? (-> env (get "RAVEN_FORCE_INGRESS_TLS" tls-default) utils/string-yes?)]
```

Comment:
Can we pass this name in the configuration?

Reply:
This is now pulled from the `raven-sidecar` config.